src/App.css

```diff
@@ -67,6 +67,14 @@ li a.active {
   padding: 1px;
 }
 
+#Footer {
+  width: 100%;
+  height: 50px;
```

Comment:
Any way to replace this height with something not as absolute as 'px' unit?
This isn't accommodating to all devices. Whenever possible use '%', 'em', 'rem' etc...